adserver/management/commands/add_advertiser.py

```diff
@@ -0,0 +1,206 @@
+"""
+Add an advertiser to the DB and setup a campaign, user, and billing.
+
+Sets up:
+
+- The Advertiser record
+- A default campaign record
+- An initial flight with some reasonable defaults
+- A user with access to the advertiser
+- If Stripe is configured, setup a Stripe customer for the advertiser
+
+This DOES NOT:
+
+- Send the newly created user an invite to the platform
+- Customize the targeting of the flight
+- Customize CPC for CPC-based advertisers
+
+Example::
+
+   ./manage.py add_advertiser -e jfoo@bigco.com -n "John Foo" -a "Big Co."
+"""
+import stripe
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+from django.core.validators import validate_email
+from django.utils.text import slugify
+from django.utils.translation import ugettext_lazy as _
+
+from ...constants import PAID_CAMPAIGN
+from ...models import Advertiser
+from ...models import Campaign
+from ...models import Flight
+from ...models import PublisherGroup
+
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+
+    """Add an advertiser from the command line."""
+
+    DEFAULT_CPM = 3.33
+    DEFAULT_NUM_IMPRESSIONS = 150000
+    DEFAULT_COUNTRY_TARGETING = [
+        # North America
+        "US",
+        "CA",
+        # Core Europe
+        "DE",
+        "GB",
+        "FR",
+        "IT",
+        "ES",
+        "CH",
+        "NL",
+        "PT",
+        "AT",
+        "BE",
+        "IE",
+        "GR",
+        "SE",
+        "DK",
+        "NO",
+        "FI",
+        # Bundled into Europe
+        "IL",
+        # Australia and New Zealand
+        "AU",
+        "NZ",
+    ]
+
+    def add_arguments(self, parser):
+        # These fields will prompt if not provided
+        parser.add_argument(
+            "-e", "--email", type=str, help=_("Email of user managing this advertiser")
+        )
+        parser.add_argument(
+            "-n", "--name", type=str, help=_("Name of user managing this advertiser")
```

Comment:
I haven't been adding these for publishers. I guess its useful if we ever want to automatically send them email from the app?

Reply:
It's slightly more important here because we use the same name on Stripe.